byway/src/main/webapp/script.css

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
Maybe change the name to base.css to be more descriptive and self-explanatory.

Reply:
changed it